cypress/integration/volunteer/generate_court_report.spec.js

```diff
@@ -12,7 +12,7 @@ context("Logging into cypress as a volunteer", () => {
     cy.get('#case-selection')
       .find('option').then(elements => {
         const option = elements[1].getAttribute('value');
-        cy.get('#case-selection').select(option);
+        cy.get('#case-selection').select(option, {force: true});
```

Comment:
This might also have been causing cypress to fail on master. This tells cypress to find this element even if it's covered by another element such as it's parent `<div class="field form-group">`

Reply:
thanks